src/Select/SelectInput.js

```diff
@@ -136,15 +136,11 @@ class SelectInput extends React.Component<AllProps, State> {
         value = child.props.value;
       }
 
-      this.props.onChange(
-        {
-          ...event,
-          target: {
-            value,
-          },
-        },
-        child,
-      );
+      event.persist();
+      // $FlowFixMe
+      event.target = { ...event.target, value };
```

Comment:
Should not use `$FlowFixMe`.  You need to implement a type refinement: https://flow.org/en/docs/lang/refinements/

That's fancy for saying you need to check first for the existence of `target`.  It is not guaranteed to be there.  Here's a link to the definition:
https://github.com/facebook/flow/blob/5c9deb498ca0b523243a8fda40160333f1f23277/lib/react.js#L408

And a flow cheatsheet (bookmark this):
https://www.saltycrane.com/flow-type-cheat-sheet/latest/

Reply:
Thanks for your help! :)
Pushed a fix, let me know if it's ok